src/App/components/Colors/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import {map} from 'lodash'
+import colors from 'package/utils/colors'
+
+const Colors = () => (
+  <div className='flex flex-wrap'>
+    {map(colors, (color) => (
+      <div 
+        key={color}
+        className='mr3 mb3 flex items-center'
+        style={{
+          minWidth: 225,
+        }}
+      >
+        <div className={`bg-${color} w2 h2 mr2`} />
+        <div>
+          {color}
+        </div>
+      </div>
+    ))}
+  </div>
+)
+
+export default Colors
```

Comment:
Maps over colors for the "Colors" screen of the app; these is also a `colorValues` module now when you need to access values in JavaScript without hard-coding them.